<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlButtonWidget</class>
 <widget class="QWidget" name="ControlButtonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AudioControlButtonWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="CustomFrame" name="customFrame">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="EM_FrameFormat" stdset="0">
      <enum>CustomFrame::EM_Normal</enum>
     </property>
     <property name="frameFormat" stdset="0">
      <string>EM_Normal</string>
     </property>
     <property name="layoutType" stdset="0">
      <string>Vertical</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>14</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="CustomToolButton" name="btnRecord">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>录制</string>
           </property>
           <property name="toolTipsText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomToolButton" name="btnPrevious">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>上一个</string>
           </property>
           <property name="toolTipsText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomToolButton" name="btnBackward">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>后退15秒</string>
           </property>
           <property name="toolTipsText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomToolButton" name="btnPlay">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>播放</string>
           </property>
           <property name="toolTipsText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomToolButton" name="btnForward">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>前进15秒</string>
           </property>
           <property name="toolTipsText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CustomToolButton" name="btnNext">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>下一个</string>
           </property>
           <property name="toolTipsText" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MusicProgressBar" name="musicProgressBar">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16</height>
          </size>
         </property>
         <property name="duration" stdset="0">
          <longLong>0</longLong>
         </property>
         <property name="clickJumpEnabled" stdset="0">
          <bool>true</bool>
         </property>
         <property name="showMilliseconds" stdset="0">
          <bool>false</bool>
         </property>
         <property name="progress" stdset="0">
          <number>0</number>
         </property>
         <property name="bufferProgress" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomFrame</class>
   <extends>QFrame</extends>
   <header>CoreWidget/CustomFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomToolButton</class>
   <extends>QToolButton</extends>
   <header>CoreWidget/CustomToolButton.h</header>
  </customwidget>
  <customwidget>
   <class>MusicProgressBar</class>
   <extends>QWidget</extends>
   <header>CoreWidget/MusicProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
